### PR TITLE
First pass use cases

### PR DESCRIPTION
**Pull request by aarongustafson**

### CODE REVIEW

File: use-cases.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Shortcuts v2 Use Cases
+
+Below is a collection of real-world uses cases provided by various organizations. They are bucketed into whether they need access to manifest-defined shortcuts (read-only or read/write) or not.
+
+## **Do** require access to manifest-defined shortcuts
+
+1. **Status:** Applications want to update specific shortcuts with up-to-date status information. Examples include
+    * a shortcut to a particular payment method (e.g., gift card) which could be updated to include the current balance.
+    * a shortcut to a user’s messaging inbox could be updated to reflect the current unread count.
```
Comment (beaufortfrancois) [inline use-cases.md:9]: @aarongustafson @rayankans I think web developers can already use the badging API for this use case. See https://web.dev/badging-api/

File: use-cases.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+2. **Localization:** Apps that do localization on the client-side want to be able to define shortcuts in the app’s default language via the manifest and then update the label given to each dynamically via the API.
```
Comment (tomayac) [inline use-cases.md:10]: Does this assume that apps also swap the default manifest for a one written in the user's language?

File: use-cases.md (hunk 1 above)
Comment (aarongustafson) [reply use-cases.md:9]: This is separate, but could be used in concert. An example might be a social media service that has both direct messages and comments. The links to each could be defined in the manifest (e.g., `/messages` and `/comments`), but the unread count would be unique for each. They could be aggregated into a badge count too, but that isn't quite as useful.

File: use-cases.md (hunk 2 above)
Comment (aarongustafson) [reply use-cases.md:10]: Based on conversations I've had with a few companies, they are not terribly thrilled with the current recommendation of having separate manifests for each language. For some, the only keys that would need translation would be the shortcuts, so having the ability to alter them after the fact would be preferable for them.

**Pull request merged**